src/test/resources/drools-kie-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
```

Comment:
I might be wrong but it seems to me that this file is not used in the end2end tests. We can remove this file, What do you think?

Reply:
This file is there because it helped to start the compose with KIE and test the commands. We can remove it , but I left it here in order to have the yaml if we need to do further tests with the compose.